components/schedule-editor/src/styles/schedule-editor.scss

```
@@ -1,6 +1,12 @@
-@import "../../../theming/reset.scss";
 @import "../../../theming/variables.scss";
 
+.sr-only {
+  top: auto;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
```

```suggestion
  top: auto;
  width: 1px;
  height: 1px;
  overflow: hidden;
  position:absolute;
  left:-10000px;
```

https://webaim.org/techniques/css/invisiblecontent/#offscreen